### PR TITLE
Update work process document

### PR DESCRIPTION
**Pull request by cas--**
The is a work in process pull request for updating and simplifying the work process document.

There are three new documents for git-conventions, scrum and issues and user stories. These mainly consist of moving sections from work-process.md to new docs and updating work-process with a link and short paragraph.

### CODE REVIEW

File: issue-user-story.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+* 100% test coverage.
+* Write/Update tech docs: are there any technical docs that need to be written or updated?
+* Write/Update support docs: notify support and they will go through the steps of the User Story and write or update the support docs.
+* Write/Update Help Center article: notify support and they will go through the steps of the User Story and write or update the Help Center.
+* Write/Update Intra/Handbook docs: notify operations and they will go through the steps of the User Story and write or update Handbook or Intra.
+* Deployed.
+
+### Expectations (Acceptance Criteria)
+
+Activities that need to be performed so this user story can be moved to `Done`. Links to PR, tested and working feature, notified support, etc.
```
Comment (zupo) [inline issue-user-story.md]: State of the world, not activities.

File: scrum.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# Scrum
+
+Sprint length: two weeks.
+
+Story Points commitment: 10 for each full-time member on the sprint and at least 10 for bugfixes and cleanup.
```
Comment (zupo) [inline scrum.md]: `~8 for each Nitean`

File: scrum.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+## Scrum on GitHub
+
+Some of ideas are taken from https://colloq.io/blog/the-tools-we-use-to-stay-afloat.
+
+Our Kanban Board is at https://github.com/niteoweb/operations/#boards. By clicking on the `Show Menu` you get an additional sidebar that displays the main information about the sprint, such as the **Sprint Goal**, our capacity for the sprint, committed story points and the **Burndown Chart**.
+
+GitHub checkboxes are often utilized in creating mini-tasks for User Stories. Since there is no history on who clicked on what checkbox and when, always add a comment about the change made everytime a checkbox is ticked.
+
+
+## Team Capacity
```
Comment (zupo) [inline scrum.md]: This entire paragraph is not needed.

Comment (cas--) [reply scrum.md]: I assume you mean Team Capacity?

**Pull request merged**